### PR TITLE
Update CHANGES.md for v1.1.0

### PR DESCRIPTION
**Pull request by stealthcode**

### CODE REVIEW

**Comment by stealthcode** [comment]
@akarnokd @stevegury @abersnaze @benjchristensen @zsxwing

**Comment by akarnokd** [comment]
:+1:

**Comment by stevegury** [comment]
:+1:

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+* 2 Observable.merge overloads for maxConcurrent
+* TestSubscriber methods
+* Observable.takeUntil with predicate 
+
+#### Promotions to BETA
+
+* ConnectableObservable.autoConnect
+* Stateful Subject methods on ReplaySubject, PublishSubject, BehaviorSubject, and AsyncSubject
+
+#### Removals from Public API
```
Comment (benjchristensen) [inline CHANGES.md]: Perhaps this should clarify that they were experimental and that's why they can be removed?

For example: "Experimental APIs Removed"

Comment (stealthcode) [inline CHANGES.md]: That would have helped. Would it help to edit the Github release and open a PR to the CHANGES.md?

Comment (benjchristensen) [inline CHANGES.md]: I think it's worth clarifying.

**Pull request merged**